src/components/GlobalAudioPlayer/TimeSliderControls.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { formatTime } from '@/lib/formatTime'
+import { TimeSliderControllerProps } from '@/types/const'
+
+const TimeSliderControls = ({
+  currentTime,
+  duration,
+  handleSliderChange
+}: TimeSliderControllerProps) => {
+  return (
```

Comment:
return statement is redundant
```suggestion
}: TimeSliderControllerProps) => (
```